docs/0.4/concepts/about_biome.md

```diff
@@ -1,17 +1,422 @@
 # About Biome
 
-Biome is a module in libsplinter containing several submodules that provide
-support for user management, user credential management, and private key
-management. These submodules, and their functionality are bundled into Biome’s
-REST API module and can be integrated into an existing splinter REST API using
-libsplinter’s rest-api feature.
+Biome is the Splinter component for user, credential, and private key
+management. It isolates and securely exposes user-based information for other
+Splinter components. Biome stores user and credential information in a local
+PostgreSQL database, manages access tokens, and provides encrypted keys for
+other Splinter actions, such as signing transactions on behalf of a user.
 
-## Modules
+Biome stores information in a local PostgreSQL database on the Splinter node.
+Splinter does not share any user, credential, or key information with other
+Splinter nodes. See [Biome Database Tables](#biome-database-tables) for the
+database structure and table names.
 
-* **User Management**: API to add, delete, update, and retrieve user
-information.
+Biome is part of the Splinter daemon, `splinterd`. The `splinterd` REST API
+exposes this functionality as Biome-specific endpoints. The [splinterd REST API
+Reference](https://www.splinter.dev/docs/0.4/api/) provides documentation and
+examples for the Biome routes and request/response format.
 
-* **Credential Management**: API to register and authenticate a user using a
-username and password. Not recommended for use in production.
+This topic explains Biome's architecture, then illustrates how Biome works with
+examples of using Biome endpoints.
 
-* **Private Key Management**: API to store and retrieve encrypted private keys.
+## Biome Architecture
+
+Biome provides the following functions:
+
+* User account management: Creates and stores users by username and ID.
+  A Splinter user has a username, access credentials, and an automatically
+  generated user ID (UUID). Biome generates the UUID when a user is registered.
+
+* Key management: Encrypts and stores users' private keys.
+
+* Credential management: Registers and authenticates users.  The initial
+  implementation uses usernames and passwords for authentication (not
+  recommended for production use).
```

Comment:
Let's clarify what exactly we do not recommend using in production.

Reply:
Let’s expand on this in a future PR

Reply:
Or maybe just say "The initial implementation uses usernames and passwords for authentication and is not recommended for production usage"?  Like we're not saying credential management in Biome isn't recommended in PROD, we're just saying the first implementation isn't.

Reply:
Made this change.